package.json

```diff
@@ -27,6 +28,7 @@
     "deploy": "gh-pages -d dist"
   },
   "dependencies": {
-    "babel-loader": "^9.1.3"
+    "babel-loader": "^9.1.3",
+    "lodash": "^4.17.21"
   }
 }
```

Comment:
## [OPTIONAL]

- [ ] Kindly make sure your app can run in development usin the `npm start` command, the npm command is giving the error below 

![Screenshot 2023-08-04 at 03 42 39](https://github.com/BiancaMe/LeaderboardProject/assets/65068771/f354ffc7-f3c1-4dc4-a0f4-3927b772ccda)

Reply:
@andersonlebon Hello, thanks for the review. When I run the npm start in my console works fine, I don't know what the problem is when you use the command.